app/lib/new_references_feature.rb

```diff
@@ -0,0 +1,16 @@
+class NewReferencesFeature
+  attr_reader :application_form
+  OLD_REFERENCE_FLOW_CYCLE_YEAR = 2022
+
+  def initialize(application_form)
+    @application_form = application_form
+  end
+
+  def active?
+    FeatureFlag.active?(:new_references_flow) && application_form.recruitment_cycle_year > OLD_REFERENCE_FLOW_CYCLE_YEAR
```

Comment:
As mentioned previously I think this should be split to use the `FeatureFlag` class and a method on `ApplicationForm` which is aware of whether the form can accept the new style of references.